src/test/transaction_tests.cpp

```diff
@@ -95,14 +92,83 @@ std::string FormatScriptFlags(unsigned int flags)
     return ret.substr(0, ret.size() - 1);
 }
 
+/*
+* Check that the input scripts of a transaction are valid/invalid as expected.
+*/
+bool CheckTxScripts(const CTransaction& tx, const std::map<COutPoint, CScript>& mapprevOutScriptPubKeys,
+    const std::map<COutPoint, int64_t>& mapprevOutValues, unsigned int flags,
```

Comment:
Nit: map_p_rev. Also, I seem to remember that new variables should preferably use snake case.

Reply:
That's true (doc/developer-notes.md)

Reply:
`flags` and `expect_valid` now passed by value, thanks :+1:

Reply:
o tru